src/compiler/haxe.ml

```diff
@@ -1098,7 +1098,7 @@ with
 			ctx.messages <- [];
 		end else begin
 			error ctx (Printf.sprintf "You cannot access the %s package while %s (for %s)" pack (if pf = "macro" then "in a macro" else "targeting " ^ pf) (s_type_path m) ) p;
-			List.iter (error ctx "    referenced here") (List.rev pl);
+			List.iter (error ctx "... referenced here") (List.rev pl);
```

Comment:
Shouldn't there be some helper function like `related_error` that handles the `... ` prefix so you can't get the formatting wrong (missing a space after or something) and it doesn't have to be repeated everywhere?

Reply:
Done